static/koschei.css

```diff
@@ -24,183 +24,6 @@
     Michael Simacek <msimacek@redhat.com>
 */
 
-html {
-    min-width: 800px;
-}
```

Comment:
Can you please add the following, so that the font sizing matches other Fedora apps that use bootstrap-fedora?
````css
html {
    font-size: 14px;
}
````